package.json

```diff
@@ -1,17 +1,18 @@
 {
   "dependencies": {
-    "@rails/webpacker": "3.5",
+    "@rails/ujs": "^6.0.2",
```

Comment:
For reference/info purposes, for maintainers or contributors, including myself: Old documentation for what Rails ujs is about: https://github.com/rails/rails-ujs/tree/master

(It's now moved to being included/developed directly in the Rails project, per: https://github.com/rails/rails-ujs/blob/moved-to-rails/README.md)